coalib/core/Core.py

```diff
@@ -2,6 +2,7 @@
 import concurrent.futures
 import functools
 import logging
```

Comment:
Is it really needed ? I thought we add blank lines b/w imports to differentiate coala modules and built-in modules.